references/cli/cluster.go

```diff
@@ -190,6 +198,18 @@ func NewClusterJoinCommand(c *common.Args, ioStreams cmdutil.IOStreams) *cobra.C
 				return err
 			}
 			cmd.Printf("Successfully add cluster %s, endpoint: %s.\n", clusterName, clusterConfig.Cluster.Server)
+
+			if len(labels) > 0 {
+				addLabels := map[string]string{}
+				for _, kv := range strings.Split(labels, ",") {
+					parts := strings.Split(kv, "=")
+					if len(parts) != 2 {
+						return errors.Errorf("invalid label key-value pair %s, should use the format LABEL_KEY=LABEL_VAL", kv)
+					}
+					addLabels[parts[0]] = parts[1]
+				}
```

Comment:
let's merge with L374-L380, they're same logic.

Reply:
That makes sense. Could you review it again?